internal/services/formatters/python/safety/formatter.go

```diff
@@ -121,7 +121,7 @@ func (f *Formatter) setupVulnerabilitiesSeveritiesSafety(
 
 func (f *Formatter) getVulnerabilityLineByName(line, fileName string) string {
 	path := filepath.Join(f.GetConfigProjectPath(), fileName)
-	fileOpened, err := os.Open(path)
+	fileOpened, err := os.Open(filepath.Clean(path))
```

Comment:
Por que isso é necessário? E não deveria estar em outro pr?

Reply:
it should in another PR, you're rigth